docs/source/release/v6.8.0/Framework/35640.rst

```diff
@@ -1 +0,0 @@
-* Two new build targets ``docs-linkcheck`` and ``dev-docs-linkcheck`` run `sphinx linkcheckbuilder <https://www.sphinx-doc.org/en/master/usage/builders/index.html#sphinx.builders.linkcheck.CheckExternalLinksBuilder>`_
```

Comment:
Did this get moved to somewhere else, or was it included in last release?

Reply:
I removed it because I don't think it was a user-facing change, and so it is not needed in the release notes